includes/admin/views/builder/video-explainer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Builder video explainer view
+ *
+ * @since [version]
+ * @version [version]
+ */
+?>
+<script type="text/html" id="tmpl-llms-video-explainer-template">
+
+	<span class="llms-video-explainer-trigger">
+		<img
+			src="<?php echo plugin_dir_url( LLMS_PLUGIN_FILE ); ?>assets/images/course-builder-video-thumbnail.jpg"
+			alt="<?php _e( 'How to Build Your Course Outline with the LifterLMS Course Builder', 'lifterlms' ); ?>"
```

Comment:
```suggestion
			alt="<?php esc_attr_e( 'How to Build Your Course Outline with the LifterLMS Course Builder', 'lifterlms' ); ?>"
```
Better escaping what comes from translations, you'll never know :D

Reply:
fc96051